docs/claims.md

```diff
@@ -0,0 +1,175 @@
+> statement. The two actions "claim" and "attest" are done at once in a `claim_attest` transaction,
+> but for the most part this is simplified from the perspective of the UI.
+
+### What you will need
+
+- The Ethereum account that holds the DOT indicator tokens.
+- The MyCrypto wallet.
+- A Polkadot account.
+
+You should already have your Ethereum account that holds the DOT indicator tokens from Polkadot's
+prior sales. You will need to have access to this account in order to make a signature.
+
+[MyCrypto][] is a versatile wallet that supports a variety of storage methods for your Ethereum
+account. Please click on the link to go to their downloads page and ensure you download the latest
+version for your operating system. This is important because the latest version will always have the
+latest security patches.
+
+You will need a Polkadot account to mark as the destination for sending the DOTs in the claim.
+Please follow the instructions on the [account generation][] page for generating a new Polakdot
+account.
+
+#### Claiming your DOT with MyCrypto
+
+The Polkadot JS [Claims app][] helps you sign a message from MyCrypto. MyCrypto is a good choice in
+case you have stored the key to the Ethereum account holding your DOT indicator tokens on a hardware
+device like a Ledger Nano S or a Trezor. It also supports raw private keys, mnemonics and Parity
+signer.
+
+**NOTICE**: It is much more secure to download and use the MyCrypto app locally. Please make sure to
+download the latest version for your operating system. You can always find the most up-to-date
+releases of the desktop app on their [releases page][mycrypto].
+
+Once you've downloaded MyCrypto and have it running locally (we recommend an air-gapped computer for
+maximum security), you can start by navigating to the Claims app on Polkadot-JS Apps. Select the
+account you would like to claim the DOTs into and click the blue "Continue" button to proceed. Your
+screen should look something like this:
+
+![claim-1](assets/new-claims/claim-1.png)
+
+Now you will need to provide the Ethereum address that is associated with the DOT indicator tokens
+that you will claim. Enter the Ethereum address into the box and click "Continue". Next your screen
+should look the image below.
+
+![claim-2](assets/new-claims/claim-2.png)
+
+The hex encoded string that follows the sentence: "Pay DOTs to the Polkadot account:" is the
+hex-encoded public key of your Polkadot account, minus the `0x` prefix. To verify that the public
+key is correct you can use the `subkey` tool to inspect your address.
+
+The next step is to go to the MyCrypto application and click on "Sign & Verify Message" tab.
+
+![claim-3](assets/new-claims/claim-3.png)
+
+This will prompt you to select a method for unlocking your wallet.
+
+![claim-3](assets/new-claims/claim-4.png)
+
+After unlocking your wallet, you will copy and paste the outputted sentence into the input box.
+
+![claim-5](assets/new-claims/claim-5.png)
```

Comment:
Additional to Amaury's comment, there is now an additional sentence added at the end (e.g for regular account it's "I hereby agree to the terms of the statement whose SHA-256 multihash is Qmc1XYqT6S39WNp2UeiRUrZichUWUPpGEThDE6dAb3f6Ny. (This may be found at the URL:  https://statement.polkadot.network/regular.html)"

Reply:
Note that some changes were made to SAFT yesterday so hash will have changed

Reply:
Indeed: https://github.com/polkadot-js/apps/pull/2734/commits/e5d7d604b53f504397e4e37532a5ba85c65cf78e